### ISSUE TITLE
findmax() on range

### ISSUE THREAD

**Issue by GregPlowman** [JuliaLang/julia#15825]
Is this a bug?
```
julia> findmax(11:20)
(20,20)

julia> findmax(collect(11:20))
(20,10)
```

similar issue for `findmin`.

**Comment by ivarne** [comment]
Looks very similar to https://github.com/JuliaLang/julia/pull/14086, but that has been fixed since 0.4.2. What version of Julia are you using?

**Comment by GregPlowman** [comment]
I apologise, I should have checked.
This does indeed seem fixed on my home computer, Julia v0.4.5.
For reasons that are embarrassingly hard to explain, my work computers are still at Julia v0.4.1.
